### ISSUE TITLE
limit builds to certain platforms

### ISSUE THREAD

**Issue by macMikey** [trevordevore/levure#212]
i don't see how i can limit builds/packages to certain platforms.
example: i've got a mobile app. i'm messing with android layouts. i want to forgo packaging for ios, and just package for android.

**Comment by trevordevore** [comment]
@macMikey Settings are taken from the Standalone Settings applied to the standalone.livecode stack. Update those (and save the stack) before packaging.

**Comment by macMikey** [comment]
right. i was hoping that i could do this without diving into the standalone builder. especially on mobile, i do a LOT of builds for testing/debugging, so constantly modifying the build platforms in the standalone settings isn't nearly as convenient as typing something like `levurepackageapplication "beta" , "android"`
it sounds like if i want to do this i'm gonna be hitting the weeds and issuing a PR
